### DISCUSSION TITLE
Expressing ignoring all lines below a "fold"

### DISCUSSION THREAD

**Discussion by rlmark** [XAMPPRocky/tokei#921] [Q&A] [answered]
Apologies if this is the wrong forum for this or if I've missed this in the docs, but when I was adding our language, [Unison](https://www.unison-lang.org/), to the tokei tool, I wasn't sure how to express a particular feature in the json configuration. In Unison, we have a feature called a "fold", demarcated by `---` which means that everything in the file including and below the line `---` should be ignored by our parser. It's a bit like a multi-line comment which has no terminal indication. Is this something that's safe to be expressed with the `"multi_line_comments"` array by omitting the second element in the array? Or is it not a currently supported feature?

**Comment by XAMPPRocky** [comment, marked as answer]
Thank you for question! Tokei doesn't currently support anything like that. Thinking about it, I would try setting the ending comment of the `multi_line_comments` to a empty string `""`, and see what results you get.
